metadata.yaml

```diff
@@ -1,25 +1,26 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-# For a complete list of supported options, see:
-# https://discourse.charmhub.io/t/charm-metadata-v2/3674/15
-name: operator-template
-display-name: |
-  TEMPLATE-TODO: fill out a display name for the Charmcraft store
+name: mysql-k8s-operator
 description: |
-  TEMPLATE-TODO: fill out the charm's description
+  MySQL RDBMS with InnoDB Cluster support for Kubernetes.
 summary: |
-  TEMPLATE-TODO: fill out the charm's summary
+  Kubernetes charm for HA MySQL InnoDB Clusters.
 
-# TEMPLATE-TODO: replace with containers for your workload (delete for non-k8s)
 containers:
-  httpbin:
-    resource: httpbin-image
+  mysql-server:
+    resource: mysql-server-image
+  mysql-shell:
+    resource: mysql-shell-image
```

Comment:
Isn't this already included in the server image?

Reply:
No, the server image has only the server binary.

Reply:
https://hub.docker.com/r/mysql/mysql-server/
https://hub.docker.com/r/ubuntu/mysql

both include the shell. are you using some other image?